oonf-olsrd2/Makefile

```diff
@@ -1,13 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=oonf-olsrd2
-PKG_RELEASE:=1
+PKG_VERSION:=v0.15.99
+PKG_REV:=1cec9b21086fb52ab4262c69aabd087e4d2d3a44
+PKG_RELEASE:=$(PKG_REV)
```

Comment:
```suggestion
PKG_RELEASE:=1
```

Reply:
Is there an easy way to get the short commit ref into the package version/release?
As this isn't a release version it will probably update in the future without a change in the version number and I think having the shortref instead of a in incrementing number (that we'll need to manually update) as the release would make more sense (kinda like it is for the olsr1 package but just entering "$(AUTORELEASE)" here did not seem to work)

Reply:
Sure, just drop ``PKG_VERSION`` and that's it. :)